<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPenjualan</class>
 <widget class="QWidget" name="FormPenjualan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="kode_PelangganLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="kode_PelangganLabel">
      <property name="text">
       <string>Kode Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="no_FakturLabel">
      <property name="text">
       <string>No Faktur</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="no_FakturLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tgl_FaktorLabel">
      <property name="text">
       <string>Tanggal Faktor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="tgl_FaktorDateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="harga_BeliLabel">
      <property name="text">
       <string>Harga Beli</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="harga_BeliLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="harga_JualLabel">
      <property name="text">
       <string>Harga Jual</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="harga_JualLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="banyak_KeluarLabel">
      <property name="text">
       <string>Banyak Keluar</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="banyak_KeluarLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="bode_BarangLabel">
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="Kode_BarangLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SImpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablePenjualan">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
